src/main/java/salesforce/endpointurl/Endpoint.java

```diff
@@ -1,4 +1,4 @@
-package endpointurl;
+package salesforce.endpointurl;
 
 public class Endpoint {
```

Comment:
Endpoints are already strings, probably there is no need to set element variables.